tests/kernel/userbufer_validate/src/userbuffer_validate.c

```diff
@@ -0,0 +1,524 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <mmustructs.h>
+#include <arch/x86/arch.h>
+#include <linker/linker-defs.h>
+#include <arch/x86/arch.h>
+#include <ztest.h>
+#include <nano_internal.h>
+
+#define SKIP_SIZE 2
+#define BUFF_SIZE 10
+
+static int status;
+
+#define BUFF_READABLE ((u32_t) 0x0)
+#define BUFF_WRITEABLE ((u32_t) 0x1)
+#define BUFF_USER ((u32_t) 0x2)
+
+int _arch_buffer_validate(void *addr, size_t size, int write);
+void reset_flag(void);
+void reset_multi_pte_page_flag(void);
+void reset_multi_pde_flag(void);
+
+#define ADDR_PAGE_1 ((void *)__bss_start + SKIP_SIZE * MMU_PAGE_SIZE)
+#define ADDR_PAGE_2 ((void *)__bss_start + (SKIP_SIZE + 1) * MMU_PAGE_SIZE)
+#define PRESET_PAGE_1_VALUE (X86_MMU_GET_PTE(ADDR_PAGE_1)->p = 1)
+#define PRESET_PAGE_2_VALUE (X86_MMU_GET_PTE(ADDR_PAGE_2)->p = 1)
+
+
+/* if Failure occurs
+ * _arch_buffer_validate return -EPERM
+ * else return 0.
+ * Below conditions will be tested accordingly
+ *
+ */
+
+/* read write testing */
+static int buffer_rw_read(void)
+{
+	PRESET_PAGE_1_VALUE;
+	_x86_mmu_set_flags(ADDR_PAGE_1,
```

Comment:
break a line after PRESET_... You do it sometime and sometime not. Apply that eveywhere

Reply:
@tbursztyka . Could you please check the style issue is addressed and fixed